src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -154,8 +156,34 @@ public override Task<Empty> Hello(Empty request, ServerCallContext context)
             {
                 try
                 {
+                    Activity? newActivity = new Activity("gRPC start orchestration");
```

Comment:
Does this span name match the convention we're using for other spans?

Reply:
I removed this new Activity.